jinglify/Game/GameSettings.swift

```diff
@@ -30,6 +30,8 @@ struct GameSettings {
             }).flatMap({ (predicate) in
                 let query = MPMediaQuery()
                 query.addFilterPredicate(predicate)
+                query.addFilterPredicate(MPMediaPropertyPredicate(value: false, forProperty: MPMediaItemPropertyHasProtectedAsset))
+                query.addFilterPredicate(MPMediaPropertyPredicate(value: false, forProperty: MPMediaItemPropertyIsCloudItem))
```

Comment:
I have several cloud tracks in my jingle list and it works OK (when internet connection is OK). Why we should forbid them?